scripts/src/java/org/oppia/android/scripts/coverage/CoverageRunner.kt

```diff
@@ -41,4 +53,176 @@ class CoverageRunner(
   ): List<String>? {
     return bazelClient.runCoverageForTestTarget(bazelTestTarget)
   }
+
+  private fun parseCoverageData(
+    coverageData: List<String>,
```

Comment:
Perhaps instead: ``coverageDataFileLines`` might be a bit clearer/more specific.

Reply:
Okay Done!